modules/dynamic-plugins-get-started.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-with-dynamic-plugins"]
+= Getting started with dynamic plug-ins
+In order to get started using the dynamic plug-in, visit this link:https://github.com/spadgett/console-plugin-template[template] containing a Dockerfile for building plug-in images.
+
+You can run the plug-in using a local development environment or build an image to deploy it to a cluster. The repository also contains a {product-title} template for adding your plug-in to the cluster.
+
+.Prerequisites
+
+* You must have administrator privileges.
+* Ensure you have `Node.js` and `yarn` installed.
+
+.Procedure
+
+. Click `Use this Template` to create a GitHub repository.
+
+. Clone the repository.
+
+. Edit the plugin metadata in `package.json`. 
```

Comment:
We should tell them specifically what they need to change (`name`, `version`, and `repository` in package.json as well as plugin information in the `consolePlugin` stanza).